### PR TITLE
feat: add hello_world service

### PR DESCRIPTION
**Pull request by dvilelaf**
## Proposed changes

Add the hello_world service.

## Fixes

n/a

## Types of changes

What types of changes does your code introduce? (A **breaking change** is a fix or feature that would cause existing functionality and APIs to not work as expected.)
_Put an `x` in the box that applies_

- [x] Non-breaking fix (non-breaking change which fixes an issue)
- [ ] Breaking fix (breaking change which fixes an issue)
- [ ] Non-breaking feature (non-breaking change which adds functionality)
- [ ] Breaking feature (breaking change which adds functionality)
- [ ] Refactor (non-breaking change which changes implementation)
- [ ] Messy (mixture of the above - requires explanation!)

## Checklist

_Put an `x` in the boxes that apply._

- [x] I have read the [CONTRIBUTING](../blob/main/CONTRIBUTING.md) doc
- [x] I am making a pull request against the `main` branch (left side). Also you should start your branch off our `main`.
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: packages/valory/services/hello_world/service.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+author: valory
+version: 0.1.0
+description: A simple demonstration of a simple ABCI application
+aea_version: '>=1.0.0, <2.0.0'
+license: Apache-2.0
+fingerprint:
+  README.md: QmPPeZtJ6aMXsGsCEynQcibxMRXypoe6K4KNAGiMi6JpaW
+fingerprint_ignore_patterns: []
+agent: valory/hello_world:0.1.0
+network: hardhat
```
Comment (dvilelaf) [inline packages/valory/services/hello_world/service.yaml:11]: Can we specify network as none?

File: packages/valory/services/hello_world/service.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+name: hello_world
```
Comment (dvilelaf) [inline packages/valory/services/hello_world/service.yaml:1]: Tried to run this but I got an error, so some config must be missing:
```
Agent hello_world successfully fetched.
Traceback (most recent call last):
  File "/usr/lib/python3.10/hashlib.py", line 160, in __hash_new
    return _hashlib.new(name, data, **kwargs)
ValueError: [digital envelope routines] unsupported
```

File: packages/valory/services/hello_world/service.yaml (hunk 1 above)
Comment (DavidMinarsch) [reply packages/valory/services/hello_world/service.yaml:11]: Not atm. Might be worth adding support for

Comment (dvilelaf) [reply packages/valory/services/hello_world/service.yaml:11]: #1001

**Review by DavidMinarsch — APPROVED** [review]

**Pull request merged**